config/activityLog.js

```diff
@@ -124,6 +124,13 @@ module.exports = {
     lastHeartbeat: 'None',
     delta: 'None',
   },
+  missedActivity: {
```

Comment:
should this log type be on by default it seemed like the more common way is to put it behind the logs toggle

Reply:
It's also on by default for the missed heartbeat logging - the idea is that this isn't something that should happen regularly so when it does we want to know about it.

Reply:
cool thanks for clearing that up